client/gutenberg/editor/main.jsx

```diff
@@ -1,12 +1,43 @@
+/** @format */
+
 /** @format */
 /**
  * External dependencies
  */
 import React, { Component } from 'react';
 
+/**
+ * Internal dependencies
+ */
+import '@wordpress/data';
+import '@wordpress/core-data';
+import { registerCoreBlocks } from '@wordpress/core-blocks';
+import { EditorProvider, WritingFlow, ObserveTyping, BlockList } from '@wordpress/editor';
+
+const editorSettings = {};
+const wpApiSettings = {
+	schema: {
+		routes: {},
+	},
+};
+const post = {
+	type: 'post',
+	content: {},
+};
+
+registerCoreBlocks();
+
 class GutenbergEditor extends Component {
```

Comment:
it looks like all you need to get started :)

Reply:
`registerCoreBlocks` needs some more love. We are considering some changes in this area before the end of July.